src/Select.js

```diff
@@ -562,6 +562,28 @@ export default class Select extends Component<Props, State> {
     });
   }
 
+  changeFocusedOption(opt: string) {
+    if(opt == null) {
+       return;
+    }
+
+    const { menuOptions } = this.state;
+    const options = menuOptions.focusable;
+    
+    if(!options.length) {
+       return;
+    }
+
+    let optionToFocus = options.find(e => e.label == opt);
+    if(!optionToFocus) {
+      return;
+    }
+
+    this.setState({
+      focusedOption: optionToFocus,
+    })
```

Comment:
In the other `focusOption` function, we are also:
1. Changing the the state of `focusValue` to null
2. Setting ` this.scrollToFocusedOptionOnUpdate = true;`
https://github.com/OutSystems/react-select/blob/main/src/Select.js#L594

Aren't they needed here also?

Reply:
1. focusedValue is used on multi select dropdowns, I had removed it while debugging, value operations change focusedOption to null, and option operations change focusedValue to null, so I will be adding it back again
2. yes, this should improve the experience on dropdowns with a bigger number of options

Reply:
Also, what does the announceAriaLiveContext that is being called in other operations? Seems to update some kind of instructions. What are we losing not calling that here?

Reply:
announceAriaLiveContext and announceAriaLiveSelection are disabled in the web-components code, with the allowScreenReaders prop